io_export_objex2/data_updater.py

```diff
@@ -191,16 +193,20 @@ def handle_material(material, layout):
     if v == addon_material_objex_version:
         return False
     if v > addon_material_objex_version:
-        layout.label(text='This material was created', icon='ERROR')
-        layout.label(text='with a newer addon version,')
-        layout.label(text='you should update the addon')
-        layout.label(text='used in your installation.')
+        box = layout.box()
+        box.label(text='', icon='ERROR')
```

Comment:
Try this maybe:
```suggestion
        box = layout.box()
        box.alert = True
        box.label(text='', icon='ERROR')
```
it should make the box red or yeloow or something like that

if it works / looks OK you can use it as you want

Reply:
That looks actually really cool! I'll add it to other error/alert places too 👍